global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version" : "2.1.401"
+    "version" : "2.1.607"
```

Comment:
Not sure about this, but sometimes in my previous Pull Requests, I had this same error build when I tried to upgrade the .NET SDK version. Appveyor and TravisCI sometimes don't support specific versions. Try changing back the version to see if it builds successfully.

Reply:
Succeeded